tools/buildtools/ambertools/parmconv.xml

```diff
@@ -0,0 +1,262 @@
+<tool id="parmconv" name="Convert Parameters" version="@VERSION@">
+  <description>to AMBER prmtop in preparation for MMPBSA</description>
+  <macros>
+    <import>macros.xml</import>
+  </macros>
+  <expand macro="requirements">
+    <requirement type="package" version="3.2.0">parmed</requirement>
+    <requirement type="package" version="2019.1">gromacs</requirement>
+    <requirement type="package" version="2.11.1">jinja2</requirement>
+  </expand>
+  <command detect_errors="exit_code">
+    <![CDATA[
+   python '$templating_script' '$inputs' &&
+   PATH_TO_PARMED=\$(dirname `which parmed`) &&
+   export AMBERHOME=\$(dirname \$PATH_TO_PARMED) &&
+   export GMXDATA=\$AMBERHOME/share/gromacs/top/ &&
```

Comment:
@chrisbarnettster, are the three lines above still needed? It looks like the variables are not used elsewhere.

Otherwise this looks good to me. :+1:

Reply:
Hi @simonbray. They are environment vars passed to parmed, and are needed by parmed to function correctly 

Awesome!

Reply:
Okay makes sense